lib/modules/datasource/golang-version/index.ts

```diff
@@ -42,7 +42,7 @@ export class GolangVersionDatasource extends Datasource {
     };
     // TODO: types (#7154)
     // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-    const golangVersionsUrl = `${registryUrl}HEAD/internal/history/release.go`;
+    const golangVersionsUrl = `${registryUrl}/HEAD/internal/history/release.go`;
```

Comment:
The handling of `registryUrl` changes when custom registries are enabled, this is addressed in discussion https://github.com/renovatebot/renovate/discussions/17807

Reply:
Please use https://github.com/renovatebot/renovate/blob/ea5806887b81b9800b941656b139e0333062f8fb/lib/util/url.ts#L8-L10 instead and revert the changes